src/clipboard.rs

```diff
@@ -0,0 +1,12 @@
+use ffi;
+use glib::translate::*;
+use Clipboard;
+
+impl Clipboard {
+    pub fn set_text(&self, text: &str) {
+        unsafe {
+            ffi::gtk_clipboard_set_text(self.to_glib_none().0, text.to_glib_none().0, text.len() as i32);
```

Comment:
Since we're passing the length, `CString` conversion can be avoided, like in [text_buffer](https://github.com/gtk-rs/gtk/blob/master/src/text_buffer.rs).